protocol/app/app.go

```diff
@@ -1000,13 +1046,16 @@ func New(
 		epochsModule,
 	)
 
+	app.ModuleManager.SetOrderPreBlockers(
+		upgradetypes.ModuleName,
+	)
+
 	// During begin block slashing happens after distr.BeginBlocker so that
 	// there is nothing left over in the validator fee pool, so as to keep the
 	// CanWithdrawInvariant invariant.
 	// NOTE: staking module is required if HistoricalEntries param > 0
 	app.ModuleManager.SetOrderBeginBlockers(
 		blocktimemoduletypes.ModuleName, // Must be first
-		upgradetypes.ModuleName,
```

Comment:
To confirm: why are we removing this? Is this because this is now part of the preblocker?

Reply:
This is part of the Cosmos upgrade guide to move it to preblocker.